definitions/syntetic/syntetic.yaml

```diff
@@ -8,14 +8,14 @@ info:
     name: Ivan Goncharov
```

Comment:
I think we should fix the folder and file name. I think it is `synthetic` with an `h`.